src/views/ModelPlan/TaskList/Basics/Milestones/index.tsx

```diff
@@ -0,0 +1,647 @@
+import { ErrorAlert, ErrorAlertMessage } from 'components/shared/ErrorAlert';
+import FieldErrorMsg from 'components/shared/FieldErrorMsg';
+import FieldGroup from 'components/shared/FieldGroup';
+import TextAreaField from 'components/shared/TextAreaField';
+import CreatePlanMilestones from 'queries/CreatePlanMilestones';
+import GetModelPlanQuery from 'queries/GetModelPlanQuery';
+import { CreatePlanMilestones as CreatePlanMilestonesType } from 'queries/types/CreatePlanMilestones';
+import {
+  GetModelPlan,
+  GetModelPlanVariables
+} from 'queries/types/GetModelPlan';
+import { UpdatePlanMilestones as UpdatePlanMilestonesType } from 'queries/types/UpdatePlanMilestones';
+import UpdatePlanMilestones from 'queries/UpdatePlanMilestones';
+import flattenErrors from 'utils/flattenErrors';
+import planBasicsSchema from 'validations/planBasics';
+
+import './index.scss';
+
+type PlanBasicsOverviewTypes = {
+  completeICIP: string | null;
+  clearanceStarts: string | null;
+  clearanceEnds: string | null;
+  announced: string | null;
+  applicationsStart: string | null;
+  applicationsEnd: string | null;
+  performancePeriodStarts: string | null;
+  performancePeriodEnds: string | null;
+  wrapUpEnds: string | null;
+  highLevelNote: string;
+  phasedIn: boolean | undefined;
+  phasedInNote: string;
+};
+
+const Milestones = () => {
+  const { t } = useTranslation('basics');
+  const { t: h } = useTranslation('draftModelPlan');
+  const { modelId } = useParams<{ modelId: string }>();
+
+  const history = useHistory();
+  const formikRef = useRef<FormikProps<PlanBasicsOverviewTypes>>(null);
+  const [hasHighLevelNote, setHasHighLevelNote] = useState(false);
+  const [hasAdditionalNote, setHasAdditionalNote] = useState(false);
+
+  const { data } = useQuery<GetModelPlan, GetModelPlanVariables>(
+    GetModelPlanQuery,
+    {
+      variables: {
+        id: modelId
+      }
+    }
+  );
+
+  const { modelName, milestones } = data?.modelPlan || {};
+
+  const [create] = useMutation<CreatePlanMilestonesType>(CreatePlanMilestones);
+  const [update] = useMutation<UpdatePlanMilestonesType>(UpdatePlanMilestones);
+
+  const initialValues: PlanBasicsOverviewTypes = {
+    completeICIP: milestones?.completeICIP ?? null,
+    clearanceStarts: milestones?.clearanceStarts ?? null,
+    clearanceEnds: milestones?.clearanceEnds ?? null,
+    announced: milestones?.announced ?? null,
+    applicationsStart: milestones?.applicationsStart ?? null,
+    applicationsEnd: milestones?.applicationsEnd ?? null,
+    performancePeriodStarts: milestones?.performancePeriodStarts ?? null,
+    performancePeriodEnds: milestones?.performancePeriodEnds ?? null,
+    wrapUpEnds: milestones?.wrapUpEnds ?? null,
+    highLevelNote: milestones?.highLevelNote ?? '',
+    phasedIn: milestones?.phasedIn ?? undefined,
+    phasedInNote: milestones?.phasedInNote ?? ''
+  };
+
+  const handleFormSubmit = (
+    formikValues: PlanBasicsOverviewTypes,
+    isAutoSave: boolean = false
```

Comment:
Same solution as mentioned in `Overview` could be used here.  And each place you call the submission sohuld be updated to reflect the props